guides/source/testing.md

```diff
@@ -1325,20 +1325,54 @@ end
 
 ### The Three Hashes of the Apocalypse
 
-After a request has been made and processed, you will have 3 Hash objects ready for use:
+**After** a request has been made and processed, you will have 3 Hash objects ready for use:
 
 * `cookies` - Any cookies that are set
 * `flash` - Any objects living in the flash
 * `session` - Any object living in session variables
 
-As is the case with normal Hash objects, you can access the values by referencing the keys by string. You can also reference them by symbol name. For example:
+```ruby
+test "POST /sessions signs in a User" do
+  alice = users(:alice)
+
+  post sign_in_path, params: { email: alice.email, password: alice.password }
+
+  assert_equal alice.id, session[:user_id]
+  assert_equal "Successfully signed in.", flash[:notice]
+end
+```
+
+As is the case with normal Hash objects, you can access the values by referencing the keys by String. You can also reference them by Symbol name. For example:
```

Comment:
I'm not sure the current style for constants, I feel like we normally don't capitalize Hash/String/Symbol/etc unless it is a Rails constant "ActiveRecord::Base", or "an Active Record object". 🤔